dask_cuda/cuda_worker.py

```diff
@@ -111,13 +111,16 @@ def del_pid_file():
         kwargs = {"worker_port": None, "listen_address": None, **kwargs}
 
         if (
-            not scheduler
-            and not scheduler_file
+            scheduler is None
+            and scheduler_file is None
```

Comment:
Just curious, is this change only to be explicit, or was there any case where the previous code would actually fail in practice?

Reply:
Just to be explicit, though I suppose if a user were to use the `CUDAWorker` constructor directly this could've led to some cryptic error messages for falsey `scheduler` or `scheduler_file` values? In practice wouldn't expect this to be a common issue

Reply:
Yeah, that makes sense. I was not considering where users could construct `CUDAWorker` directly, although I don't think it exists in practice today. Anyway, was just asking to know if you had encountered something already, seems like we're on the same page. 🙂